deployer/scripts/upgrade.sh

```diff
@@ -513,6 +514,14 @@ function regenerate_secrets_and_support_objects() {
   generate_support_objects
 }
 
+function remove_triggers() {
+
+  for dc in $(oc get dc -l logging-infra -o jsonpath='{.items[?(@.spec.triggers[*].type)].metadata.name}'); do
+    oc patch dc/$dc -p='{"spec": { "triggers" : [] } }'
+    oc deploy dc/$dc --latest
```

Comment:
this is true, i think i was doing this because i wasn't sure if just removing it would unregister the dc triggers; should be fine though, will remove